module_utils/network_lsr/nm/active_connection.py

```diff
@@ -6,11 +6,8 @@
 
 # Relative import is not support by ansible 2.8 yet
 # pylint: disable=import-error, no-name-in-module
-from ansible.module_utils.network_lsr.nm.client import GLib  # noqa:E501
-from ansible.module_utils.network_lsr.nm.client import NM  # noqa:E501
-from ansible.module_utils.network_lsr.nm.client import get_mainloop  # noqa:E501
-from ansible.module_utils.network_lsr.nm.client import get_client  # noqa:E501
-from ansible.module_utils.network_lsr.nm.error import LsrNetworkNmError  # noqa:E501
+from ansible.module_utils.network_lsr.nm import client  # noqa:E501
+from ansible.module_utils.network_lsr.nm import error  # noqa:E501
```

Comment:
You might not need these `  # noqa:E501` if the line length is short enough now

Reply:
I think I added them since they were needed in the collection converted files...